Sources/SteamPress/Controllers/BlogController.swift

```diff
@@ -30,12 +30,20 @@ struct BlogController {
         drop.group(pathCreator.blogPath ?? "") { index in
             index.get(handler: indexHandler)
             index.get(blogPostsPath, String.self, handler: blogPostHandler)
-            index.get(tagsPath, String.self, handler: tagViewHandler)
-            index.get(authorsPath, String.self, handler: authorViewHandler)
             index.get(apiPath, tagsPath, handler: tagApiHandler)
             index.get(blogPostsPath, handler: blogPostIndexRedirectHandler)
-            index.get(tagsPath, handler: allTagsViewHandler)
-            index.get(authorsPath, handler: allAuthorsViewHandler)
+            
+            let disabledPaths = config.disabledPaths
```

Comment:
I agree with you against checking using strings. I guess the `Bool` alternative will be better fow now since there are still 2 of them.